cmd/restic/cmd_find_integration_test.go

```diff
@@ -91,3 +91,53 @@ func TestFindJSON(t *testing.T) {
 	rtest.Assert(t, len(matches[0].Matches) == 3, "expected 3 files to match (%v)", matches[0].Matches)
 	rtest.Assert(t, matches[0].Hits == 3, "expected hits to show 3 matches (%v)", datafile)
 }
+
```

Comment:
Please run `gofmt` on the file. Ideally, set up your editor to automatically format the file when saving it. For example, in VS Code the Go extension even automatically enables auto formatting of files. Thankfully Go has a canonical formatting style so there's no more need to manually format and indent code.